scour/scour.py

```diff
@@ -822,34 +823,44 @@ def renameID(doc, idFrom, idTo, identifiedElements, referencedIDs):
                     node.setAttribute(attr, newValue)
                     num += len(oldValue) - len(newValue)
 
-        del referencedIDs[idFrom]
-        referencedIDs[idTo] = referringNodes
+        # We deliberately leave referencedIDs alone to enable us to bulk update
+        # IDs where two nodes swap IDs.  (GH#186)
+        # del referencedIDs[idFrom]
+        # referencedIDs[idTo] = referringNodes
 
     return num
 
 
+def _protectedIDs(seenIDs, options):
```

Comment:
You should name this `protected_ids` for consistency with the other function (which I feel is more important than to mark this internal, for which the long-term goal is something along the lines of #121).

Also note we try to use PEP8 naming for new code where reasonable (don't force it though - only if it makes sense and doesn't introduce any inconsistencies). Naming similar things similarly is always more important than following any guidelines.